File: 8_busqueda_secuencial.cs
using System;
class Program{
    static int Secuencial(int[] arr, int val){
        for(int i=0; i<arr.Length; i++) if(arr[i]==val) return i;
        return -1;
    }
    static void Main(){
        Console.WriteLine(Secuencial(new int[]{10,20,30,40},30));
    }
}

File: 8_busqueda_secuencial.java
public class Main {
    static int secuencial(int[] arr, int val){
        for(int i=0;i<arr.length;i++) if(arr[i]==val) return i;
        return -1;
    }
    public static void main(String[] args){
        System.out.println(secuencial(new int[]{10,20,30,40},30));
    }
}